### PR TITLE
Refactoring user_can_vote method.

### PR DESCRIPTION
**Pull request by victor-torres**

### CODE REVIEW

**Comment by wagnerluis1982** [comment]
Why set coding? I didn't see any utf8 char

**Comment by victor-torres** [comment]
@wagnerluis1982 I think we should always define the file encoding to avoid conflicts. 

In this case we have another reason that is the `’` (apostrophe) character.

https://www.python.org/dev/peps/pep-0263/

**Comment by wagnerluis1982** [comment]
Ok, but in this case, `’` could be replaced by the ascii `'`.

**Comment by victor-torres** [comment]
> Single quotes are only used around words - they come in pairs, and are not part of any word.

http://english.stackexchange.com/questions/36046/apostrophe-vs-single-quote

**Comment by wagnerluis1982** [comment]
Hi, I'm working on issue #91 and this PR changes the same method that I need.

Although the unnecessary utf8 apostrophe, I think the changes here are in good shape.

**Comment by victor-torres** [comment]
@wagnerluis1982, it’s a good practice to maintain a development branch updated with the master.

**Comment by mauricioabreu** [comment]
@victor-torres @wagnerluis1982 yes, it is! But to avoid more commits (unnecessary) in the history I think you want `rebase` instead of `merge`.

**Comment by wagnerluis1982** [comment]
@mauricioabreu that was my intent to say, I always `rebase` my code when the master has updates.

File: deck/models.py [hunk 1]
```diff
@@ -209,17 +210,23 @@ def user_already_voted(self, user):
...
-           (self.author == user and not self.event.author == user):
-            pass
-        elif self.event.allow_public_voting:
-            can_vote = True
-        elif user.is_superuser:
-            can_vote = True
-        elif self.event.jury.users.filter(pk=user.pk).exists():
-            can_vote = True
-        return can_vote
+        if self.user_already_voted(user):
```
Comment (luanfonceca) [inline deck/models.py]: for me, i prefer the way that we have a local boolean var, like we had before this changes, but this works fines too and i am merging it

o/

**Pull request merged**